engine/graphics/src/vulkan/graphics_vulkan.cpp

```diff
@@ -943,7 +945,6 @@ namespace dmGraphics
 
         uint16_t validation_layers_count;
         const char** validation_layers = GetValidationLayers(&validation_layers_count, context->m_UseValidationLayers, context->m_RenderDocSupport);
-        Texture* resolveTexture = new Texture;
```

Comment:
Something happens with the texture constructor when you link with both vulkan and opengl adapters due to symbol naming. Both types are defined as dmGraphics::Texture so I'm guessing the linker just picks the first one it can find when both libraries are linked.. This needs some investigation maybe, but to fix the issue right now we avoid calling the constructor..